public/app/features/alerting/unified/utils/constants.ts

```diff
@@ -1,6 +1,6 @@
 export const RULER_NOT_SUPPORTED_MSG = 'ruler not supported';
 
-export const RULE_LIST_POLL_INTERVAL_MS = 20000;
+export const RULE_LIST_POLL_INTERVAL_MS = 60000;
```

Comment:
How do you feel about changing this? I can hide it behind the feature toggle but 20 seconds is often too short especially for larger setups

Reply:
Let's try 30sec